doc/classes/RandomNumberGenerator.xml

```diff
@@ -5,6 +5,12 @@
 	</brief_description>
 	<description>
 		RandomNumberGenerator is a class for generating pseudo-random numbers. It currently uses PCG32. The underlying algorithm is an implementation detail. As a result, it should not be depended upon for reproducible random streams across Godot versions.
+		To generate a random float number (within a given range):
+		[codeblock]
+		var rng = RandomNumberGenerator.new()
+		func _process(delta):
```

Comment:
I don't think it should run in `_process`, for an example retrieve a new random number 60 times per second seems a bit overkill.

I would suggest this instead:
```gdscript
var rng = RandomNumberGenerator.new()
func _ready():
    rng.randomize()
    var my_random_number = rng.randf_range(-10.0, 10.0)
```
Or, if we want to avoid people thinking that they need to randomize each time they need a new number from the RNG, it could be:
```gdscript
var rng = RandomNumberGenerator.new()

func _ready():
    rng.randomize()

func get_random_float():
   return my_random_number = rng.randf_range(-10.0, 10.0)
```

Reply:
Cool. I think I will update it with your first codeblock, because I think it's a bit easier and simpler to understand how simple it is to use the RNG class from a beginner's perspective